osquery/filesystem/posix/fileops.cpp

```diff
@@ -140,6 +147,11 @@ Status PlatformFile::isExecutable() const {
   return Status(1, "Not executable");
 }
 
+// Stub'd out for now
```

Comment:
Is there an issue to hash this out? Can we make one and add the number to this comment?